src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -58,6 +58,11 @@ class ConfigurationProvider implements ConfigurationProviderInterface
      */
     protected const DEFAULT_IS_PR_AUTO_MERGE_ENABLED = false;
 
+    /**
+     * @var bool
+     */
+    protected const NO_INSTALL_PACKAGES = false;
```

Comment:
`COMPOSER_INSTALL_DEPENDENCIES`
Afaik, composer should use `--no-install` option by default.

Reply:
I would keep existing behavior by default. but ok